WoW-Realm-Tracker/Models/Realm.swift

```diff
@@ -1,27 +1,36 @@
 import Runes
 import Argo
+import Curry
 
 struct Realm {
-    let name: String
-    let type: String
-    let status: Bool
+    enum RealmType: String, CustomStringConvertible {
+        case PvP = "pvp"
```

Comment:
That's what I wanted to do originally. Unfortunately, the values are coming in over the API in all lowercase.